pallets/avn-anchor/src/lib.rs

```diff
@@ -81,6 +80,22 @@ pub mod pallet {
     #[pallet::getter(fn next_chain_id)]
     pub type NextChainId<T> = StorageValue<_, ChainId, ValueQuery>;
 
+    #[pallet::storage]
+    #[pallet::getter(fn checkpoints)]
+    pub type Checkpoints<T: Config> = StorageDoubleMap<
+        _,
+        Blake2_128Concat,
+        ChainId,
+        Blake2_128Concat,
+        CheckpointId,
+        H256,
+        ValueQuery,
+    >;
+
+    #[pallet::storage]
+    #[pallet::getter(fn next_checkpoint_id)]
+    pub type NextCheckpointId<T> = StorageValue<_, CheckpointId, ValueQuery>;
```

Comment:
Do we want this to be global to the pallet? Doesn't it make more sense it it was scoped to the ChainId?